doc/spec_exceptionHandling.dox

```diff
@@ -49,48 +51,50 @@ namespace ChimeraTK {
-      - \anchor exceptionHandling_b_3_1_2_1 3.1.2.1 In a recovery group, DeviceManagers wait until all involved DeviceManagers complete the register value restoring before activating the asynchronous read in B.3.1.3 ("barrier POST-WRITE-RECOVERY"). [link to test]
-      - \anchor exceptionHandling_b_3_1_2_2 3.1.2.2 If any DeviceManager sees an exception while restoring register values, *all* DeviceManagers in the recovery group restart the recovery procedure after the POST-WRITE-RECOVERY barrier.
-    - \anchor exceptionHandling_b_3_1_3 3.1.3 The asynchronous read transfers of the device are (re-)activated by calling Device::activateAsyncReads(). [\ref testExceptionHandling_b_3_1_3 "T"]
-    - \anchor exceptionHandling_b_3_1_4 3.1.4 Finally, \c Devices/\<alias\>/deviceBecameFunctional is written to inform any module subscribing to this variable about the finished recovery. \ref comment_b_3_1_4 "(*)" [\ref testExceptionHandling_b_3_1_4 "T"]
-  - \anchor exceptionHandling_b_3_2 3.2 Any number of initialisation handlers can be added to the DeviceManager in the user code via the DeviceModule. Initialisation handlers are callback functions which will be executed when a device is opened for the first time and after a device recovers from an exception, before any application-initiated transfers are executed (including delayed write transfers). See DeviceModule::addInitialisationHandler(). [\ref testExceptionHandling_b_3_2 "T"]
-  - \anchor exceptionHandling_b_3_3 3.3 The application terminates cleanly, even if the recovery is waiting at one of the barriers mentioned in \ref b_3_1 "3.1" [link to test]
+  - \anchor exceptionHandling_b_3_1 3.1 In a recovery group (see \ref exceptionHandling_a_5_1 "A.5.1"), DeviceManagers wait until all involved DeviceManagers have seen the error condition before trying to re-open ("barrier POST-DETECT"). [\ref exceptionHandling_b_3_1 "T" ]
+  - \anchor exceptionHandling_b_3_2 3.2 After successfully re-opening the device, a recovery procedure is executed before allowing any read/write operations from the ApplicationModules and FanOuts again. This recovery procedure involves the following steps:
+    - \anchor exceptionHandling_b_3_2_1 3.2.1 In a recovery group, DeviceManagers wait until all involved DeviceManagers successfully complete the open step before starting the initialisation handler in B.3.2.2 ("barrier POST-OPEN"). [\ref testExceptionHandling_b_3_2_1 "T"]
+    - \anchor exceptionHandling_b_3_2_2 3.2.2 The so-called initialisation handlers are executed (see \ref exceptionHandling_b_3_3 "3.3") [\ref testExceptionHandling_b_3_2_2 "T"].
+      - \anchor exceptionHandling_b_3_2_2_1 3.2.2.1 The device is closed before the initialisation handler is called, and reopened afterwards. This allows devices which can only be opened once to use external init handler scripts. \ref exceptionHandling_comment_b_3_2_2_1 "(*)" [\ref testExceptionHandling_b_3_2_2_1 "T"]
```

Comment:
This is a breaking change. The commit "feat: close device before calling init handler" should hence be titled "feat!: close device before calling init handler" to mark the commit as breaking change, ideally with some additional explanation in the commit body (only apps directly providing a C++ init handler are affected, not those which use the scripted init handler).

Reply:
done